src/main/java/core/basesyntax/service/ReportToCsv.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.StockDao;
+import core.basesyntax.db.StockDaoStorageImpl;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportToCsv implements Report {
+    private static final String FILE_NAME = "report.csv";
+    private final StockDao stockDao = new StockDaoStorageImpl();
+
+    @Override
+    public void prepare() {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(FILE_NAME));

```

Comment:
Consider using a try-with-resources statement for `BufferedWriter` to ensure that the resource is closed automatically, even if an exception occurs. This will help prevent resource leaks.